src/main.ts

```diff
@@ -227,6 +235,26 @@ function toClosureJS(
       let {output} = tsickle.processES5(
           fileName, moduleId, jsFiles.get(fileName)!, cliSupport.pathToModuleName);
       jsFiles.set(fileName, output);
+    } else {
+      const tscSourceMapJson: any = jsFiles.get(fileName);
```

Comment:
new SourceMapConsumer() takes in either a string or an object, but the d.ts only has the object version.